grammars/php.cson

```diff
@@ -487,233 +480,506 @@
         ]
       }
       {
-        'begin': '(?=<<<\\s*(\'?)([a-zA-Z_]+[a-zA-Z0-9_]*)(\\1)\\s*$)'
+        'begin': '(?=<<<\\s*\'([a-zA-Z_]+[a-zA-Z0-9_]*)\'\\s*$)'
         'end': '(?!\\G)'
-        'name': 'string.unquoted.heredoc.nowdoc.php'
+        'name': 'string.unquoted.nowdoc.php'
         'patterns': [
           {
-            'include': '#heredoc_interior'
+            'include': '#nowdoc_interior'
           }
         ]
       }
     ]
-    'repository':
-      'heredoc_interior':
+  'heredoc_interior':
+    'patterns': [
+      {
+        'begin': '(<<<)\\s*("?)(HTML)(\\2)(\\s+)?$'
```

Comment:
One last thing.  I believe that the `(\\s+)?` can be changed to simply `(\\s*)` - can you confirm?